libs/features/mas/src/merch-card.js

```diff
@@ -57,6 +59,14 @@ function registerPriceOptionsProvider(masCommerceService) {
     masCommerceService.providers.price(priceOptionsProvider);
 }
 
+const intersectionObserver = new IntersectionObserver((entries) => {
+  entries.forEach((entry) => {
+    if (entry.target.clientHeight === 0) return;
```

Comment:
in some cases, I see that height sync is still not working in the tabs. Can you remove the link `if (entry.target.clientHeight === 0) return;`  and try again?
I would be ok to add a setTimeout before `entry.target.requestUpdate();` if it persists.

Reply:
this is a not blocker for me.